### DISCUSSION TITLE
vectorsService.upsertVector - PasswordException: No password given

### DISCUSSION THREAD

**Discussion by ayanna-africa** [FlowiseAI/Flowise#2534] [Q&A]
Hey guys,
I had an issue with flowise when I tried to upsert a protected pdf.

`{"statusCode":500,"success":false,"message":"Error: vectorsService.upsertVector - PasswordException: No password given","stack":{}}`

Do you know how can I add the password to unlock the file?
I tried to add the password to the metadata but it still not working...
Do you have any idea to fix this issue?
Thanks in advance

**Comment by tarpediem** [comment]
same issue
